src/test/java/de/tum/cit/aet/artemis/communication/notifications/service/ConversationNotificationServiceTest.java

```diff
@@ -125,4 +130,26 @@ void createNotificationForNewMessageInConversation() {
         conversationParticipantRepository.deleteAllById(participants.stream().map(ConversationParticipant::getId).toList());
         conversationRepository.deleteAllById(List.of(oneToOneChat.getId()));
     }
+
+    // This caused a bug in notifications on the mobile apps, see https://github.com/ls1intum/artemis-android/issues/391
+    @Test
+    @WithMockUser(username = TEST_PREFIX + "student1", roles = "USER")
+    void createNotificationForNewMessageInGroupChatContainsNonNullGroupName() {
```

Comment:
Small nitpick from the [docs](https://docs.artemis.cit.tum.de/dev/guidelines/server-tests.html#general-testing-tips):

> We want to follow the naming convention should<ExpectedBehavior>When<StateUnderTest>.

```suggestion
    void shouldContainNonNullGroupNameInNotificationWhenCreatingNewMessageInGroupChat() {
```